### PR TITLE
Release 2.3.0

### PR DESCRIPTION
**Pull request by rannick**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/megafusion.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python
```
Comment (maxulysse) [inline bin/megafusion.py:1]: Can we have more info about where this script is coming from in comments around here?

Comment (maxulysse) [reply bin/megafusion.py:1]: + Licensing would be amazing

Comment (rannick) [reply bin/megafusion.py:1]: I agree the script is not yet optimal, function definition and comments have to be added. It is meant as the first version. More functionalities and better documentation will be added (issue https://github.com/nf-core/rnafusion/issues/356)

**Review by maxulysse — APPROVED** [review]

**Review by jemten — APPROVED** [review]
The code looks good to me. I assume that you have tested on a few "real" samples :) ?

**Comment by rannick** [comment]
Yes!

**Pull request merged**